javascript/javascript2/week1/SpiritAnimal.css

```diff
@@ -0,0 +1,29 @@
+.myForm {
+  border: 2cm;
+  background-color: rgb(2, 48, 33);
+  height: 200px;
+  width: 300px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+label {
+  /* In order to define widths */
+
+  width: 85%;
+}
+
+.nameLabel {
+  font-size: 20px;
```

Comment:
For font sizes, it is recommended to use points (pt), similar to Word, Excel etc. 
As an alternative, you can use `em`, which defines the font size relative to the page's font size. For example, if you set a heading to `2em`, it will be twice as large the as ordinary text on the page. 
You can read more here: https://fonts.google.com/knowledge/glossary/em